app/controllers/organizations/staff/dashboard_controller.rb

```diff
@@ -1,11 +1,60 @@
 class Organizations::Staff::DashboardController < Organizations::BaseController
+  before_action :context_authorize!, only: %i[index incomplete_tasks overdue_tasks]
+  include Pagy::Backend
   layout "dashboard"
 
   def index
     @user = current_user
     @organization = Current.organization
     @hide_footer = true
+    @not_completed_not_overdue_tasks_count = Task.is_not_completed.not_overdue.count
+    @not_completed_overdue_tasks_count = Task.is_not_completed.overdue.count
+  end
+
+  def incomplete_tasks
```

Comment:
Does this instance variable get used? @organization

Reply:
My bad .. it was used earlier but not required now. Have removed and pushed code.